src/liboslexec/oslexec_pvt.h

```diff
@@ -720,6 +718,12 @@ class ShadingSystemImpl {
         return m_closure_registry.get_entry(id);
     }
 
+    /// Attributes to control GPU optimization
+    bool gpu_no_inline() const { return m_gpu_no_inline; }
```

Comment:
That is a good question. We could specify `cuda` or `optix` rather than `gpu`. There might be things we'd want to do differently depending on whether OptiX or CUDA was the target. I'm sure there will be greater differences between hardware vendors.

Reply:
@lgritz for ShadingSystem attributes, would you like to specify a vendor specific "extension" (well probably best as prefix) for attributes. 
@tgrant-nv not sure which applies best in your case pxt_no_inline vs. cuda_no_inline, optix_no_inline, nvidia_no_inline.

Reply:
Let's default to "optix_blah" for the things that are specific to the OptiX path. (I'll let Tim make the call about whether, when, or where it might be better to distinguish cuda vs optix. I'm not sure we currently have a working "pure Cuda" path at the moment, so it may be a moot point at present.)

Alex's point is well taken -- in addition to not assuming one "vendor" is equivalent to all GPU, we can easily imagine a future code path that is for traditional GPU rasterization (e.g. glsl or the like) that is quite different from the current optix path (even for Nvidia cards) and could lead to confusion among future developers. GPU is probably just too generic, since there are so many different and mutually exclusive GPU approaches.

Reply:
Excellent points. I have updated the prefixes for those attributes.